executing/_pytest_utils.py

```diff
@@ -0,0 +1,16 @@
+import sys
+
+
+
+def is_pytest_compatible() -> bool:
```

Comment:
Do you want to use this function outside of executing? Is it worth exposing publicly?

Reply:
I know that I need this in inline-snapshot and I would need to define it there otherwise. I don't think that it is a good idea to have to implement this in other libraries because the conditions are based on internals of executing which might also change in the future.